### PR TITLE
[CHECKBOX][SWE-1673] Write checkbox component

### PR DESCRIPTION
**Pull request by Olexandr25**
Screenshot 2022-07-28 at 18 23 50

### CODE REVIEW

File: src/app/components/elements/Checkbox/Checkbox.styled.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  -webkit-appearance: none;
+  cursor: pointer;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-right: var(--mr);
+  transition: 0.3s;
+  width: 1rem;
+  height: 1rem;
+  border-radius: var(--br-radius);
```
Comment (alexpassss) [inline src/app/components/elements/Checkbox/Checkbox.styled.js:28]: br-radius??

File: src/app/components/elements/Checkbox/Checkbox.styled.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  font-weight: var(--brandbook-module-title-fw-medium);
+  color: var(--brandbook-module-neutral);
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+
+  ${({ variant }) =>
+    variant === 'disabled' &&
+    css`
+      color: rgba(var(--neutral), var(--alpha-4));
```
Comment (alexpassss) [inline src/app/components/elements/Checkbox/Checkbox.styled.js:52]: Should be moved to the checkbox related var

File: src/app/components/elements/Checkbox/Checkbox.styled.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  }
+
+  ${({ variant }) => checkboxUnChecked(variant)}
+
+  :checked {
+    ${({ variant }) => checkboxChecked(variant)}
+  }
+`
+export const CheckMark = styled.label`
+  font-size: var(--brandbook-module-text-body1-fs);
```
Comment (alexpassss) [inline src/app/components/elements/Checkbox/Checkbox.styled.js:41]: Fix var naming

File: src/app/components/elements/Checkbox/Checkbox.vars.module.css [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    --alpha-3: 0.4;
+    --alpha-4: 0.32;
+    --alpha-5: 0.2;
+    --alpha-6: 0.05;
+
+    /* Unchecked - bg */
+    --disabled-bg: rgba(var(--neutral), var(--alpha-6)); /* variant="disabled" /*
+
+    /* Checked - bg */
+    --default-checked-bg:  rgba(var(--accent), var(--alpha-1)); /* 1 */
```
Comment (alexpassss) [inline src/app/components/elements/Checkbox/Checkbox.vars.module.css:17]: Where is mediator?

File: src/app/components/elements/Checkbox/Checkbox.vars.module.css [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    --hover-checked-bg: rgba(var(--accent), var(--alpha-2)); /* 0.72 */
```
Comment (alexpassss) [inline src/app/components/elements/Checkbox/Checkbox.vars.module.css:18]: Everything related to the colors should be inside global theme.css

**Review by alexpassss — CHANGES_REQUESTED** [review]

File: src/app/components/elements/Checkbox/Checkbox.styled.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+  :checked {
+    ${({ variant }) => checkboxChecked(variant)}
+  }
+`
+export const CheckMark = styled.label`
+  display: flex;
+  cursor: pointer;
+  align-items: center;
+  color: var(--checkbox-text-color);
```
Comment (alexpassss) [inline src/app/components/elements/Checkbox/Checkbox.styled.js:45]: --checkbox-color

File: src/app/components/elements/Checkbox/Checkbox.vars.module.css [hunk 7]
```diff
@@ -0,0 +1,31 @@
+:root {
+    /* Background */
+    --checkbox-checked-bg-default: var(--action-accent-default);
```
Comment (alexpassss) [inline src/app/components/elements/Checkbox/Checkbox.vars.module.css:3]: everything related to the colors should be moved to the theme file

**Comment by Olexandr25** [comment]
Screenshot 2022-08-01 at 12 11 53

**Review by alexpassss — APPROVED** [review]

**Pull request merged**